static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-pixel.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import config from 'lib/config';
+
+export const twitterPixel: () => ThirdPartyTag = () => ({
+    shouldRun: config.get('switches.twitterTrackingPixel', false),
+    url: '',
+    useImage: true,
+});
```

Comment:
Why is this added in here? Is this something that was part of Krux and needed keeping?